engine/lib/model/typedefs.go

```diff
@@ -24,6 +24,8 @@ type (
 	WorkerID = string
 	// Epoch is an increasing only value
 	Epoch = int64
+	// JobID is the alias of MasterID
+	JobID = MasterID
```

Comment:
This code makes me confused because the `MasterID` is an alias of `string` and we make `JobID` as an alias of `MasterID`. Do you want both `JobID` and `MasterID` to be the same type as `string`?

Reply:
Yes. They need to be the same type as `string`.
`JobID` is some kind of outer concept for user.
`MasterID` is the inner concept for framework schedule。